lib/stagehand/staging/controller.rb

```diff
@@ -9,8 +9,11 @@ module Controller
       end
 
       # Creates a stagehand commit to log database changes associated with the given record
-      def stage_changes(subject_record = nil, &block)
-        Staging::Commit.capture(subject_record, &block)
+      def stage_changes(*args, &block)
+        options = args.extract_options!
+        subject_record = args.first
+
+        Staging::Commit.capture(subject_record, options, &block)
```

Comment:
Let's just pass `*args` straight in, no need to separate them first. Or is there?